src/applications/disability-benefits/4142/pages/facilityAddress.jsx

```diff
@@ -0,0 +1,182 @@
+import _ from 'lodash/fp';
```

Comment:
Is this file being used anywhere?

Reply:
There are a couple of `_.merge`'s in the `uiSchema`, so yeah.